recipes/slicer/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "slicer" %}
+{% set version = "0.0.3" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/slicer-{{ version }}.tar.gz
+  sha256: c82500e0120e648dcd541f785e1c04a35fa9cd61baed5cf3f83e6b6b21667114
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - python >=3.6
+
+test:
+  imports:
+    - slicer
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/interpretml/slicer
```

Comment:
This URL doesn't exist

Reply:
Sorry for the delay! I spent the past few days working with the team to get this from private to public. It now exists :) ...are there any other blockers I should look into?

Thanks!